src/plugins/discover/public/application/angular/discover.html

```diff
@@ -101,45 +101,17 @@ <h1 class="euiScreenReaderOnly">{{screenTitle}}</h1>
               class="dscTimechart"
               ng-if="opts.timefield"
             >
-              <header class="dscTimechart__header">
-                <div class="small">
-                  <label
-                    for="dscResultsIntervalSelector"
-                    tooltip="{{::'discover.howToChangeTheTimeTooltip' | i18n: {defaultMessage: 'To change the time, click the calendar icon in the navigation bar'} }}"
-                  >
-                    {{toMoment(timeRange.from)}} - {{toMoment(timeRange.to)}}
-                  </label>
-
-                  &mdash;
-
-                  <span class="form-inline">
-                    <select
-                      id="dscResultsIntervalSelector"
-                      class="dscResults__interval form-control"
-                      ng-model="state.interval"
-                      ng-options="interval.val as interval.display for interval in intervalOptions | filter: intervalEnabled"
-                      ng-blur="toggleInterval()"
-                      data-test-subj="discoverIntervalSelect"
-                    >
-                    </select>
-                    <span ng-if="bucketInterval.scaled">
-                      <icon-tip
-                        content="getBucketIntervalToolTipText()"
```

Comment:
Couldn't see this tooltip, I have replaced it with EuiIconTip (icon warning). Is it ok?